### ISSUE TITLE
Out-of-sample projections

### ISSUE THREAD

**Issue by lisitsyn** [lisitsyn/tapkee#9]

**Comment by skn123** [comment]
Any updates on this?

**Comment by lisitsyn** [comment]
@skn123 unfortunately, no, any specific request?

**Comment by skn123** [comment]
I think a Laplacian eigenmap OOS would be useful. Without OOS, most of these methods are not useful (other than linear ones)

**Comment by lisitsyn** [comment]
@skn123 yeah, they are more for exploration without OOS. Can't promise anything but will try to get some time/power to do that.
